docs/DEVELOPING.md

```diff
@@ -612,6 +612,118 @@ $ make ftest NAME=oracle
 
 ℹ️ The e2e test uses default values defined in [Configure Oracle Database connector](#configure-oracle-database-connector)
 
+## Microsoft SQL Connector
+
+The [Elastic Microsoft SQL connector](../connectors/sources/mssql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+Microsoft SQL Server versions 2017, 2019, and Azure SQL, Amazon RDS for Microsoft SQL Server are compatible with Elastic connector frameworks.
+
+**Prerequisites**
+
+- The Microsoft SQL connector requires an ODBC driver to execute. Connector supports `Microsoft ODBC Driver 18`. To install ODBC driver please refer following [documentation](https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-driver-manager).
```

Comment:
This is changing in https://github.com/elastic/connectors-python/pull/630. I think it's fine to temporarily backport this as is though, and backport the update from 630 when it's ready.